backend-volto/scripts/create_site.py

```diff
@@ -45,10 +44,11 @@ def asbool(s):
 payload = {
     "title": "Plone 6 Demo Site",
     "profile_id": _DEFAULT_PROFILE,
-    "extension_ids": ["plone6demo:default", "plone6demo:initial", "plone.volto:demo"],
+    "distribution_name": "default",
+    "extension_ids": ["plone6.demo:default", "plone6.demo:initial", "plone.volto:demo"],
```

Comment:
extension_ids is ignored if distribution_name is set. These need to either be dependencies of plone6.demo:default, or installed separately below.

Reply:
Done.